Interactable.podspec

```diff
@@ -14,7 +14,7 @@ Pod::Spec.new do |s|
   s.platform     = :ios, "7.0"
 
   s.source       = { :git => "https://github.com/wix/react-native-interactable.git" }
-  s.source_files  = "lib/ios/Interactable/*.{h,m}"
+  s.source_files  = "ios/Interactable/*.{h,m}"
```

Comment:
As someone unfamiliar with `.podspec` I had to check at https://guides.cocoapods.org/syntax/podspec.html to assure myself that these curly braces weren't a problem here.

Reply:
Yeah, other project actually tend to use `ios/**/*.{h,m}`.